content/departments/technical-success/ta/ta-playbooks/ta-scenarios.md

```diff
@@ -0,0 +1,85 @@
+- These traits are explained in more detail [here](../team-culture/working-with-customers.md#customer-health)
+
+## My customer has a low TA Pulse:
+
+- If you're coming here then more often than not you did not assign this TA pulse. It is important to understand from any previous account team why there may have been concern if it is not evident from their current health dashboard. TA pulse is subjective and can be related to other health metrics, or concern for risk in the future.
+
+## My customer has a low Technical Health Score / Last Month Versions Behind:
+
+- Review notes to see if they have deployment challenges.
+- Pull information about the biggest improvements from their current version and our latest release.
+- Relate those improvements back to their engineering goals.
+- Ensure they are aware of multi-version upgrades.
+- Evaluate the deployment and see if they are set up for long-term success. A deployment change might be useful based on our updated methods.
+- Relay the benefits of precise code intelligence and understand any barriers to deploying executors.
+- Reach out to Implementation Engieering for support with upgrading/deploying executors.
+- Sync with the AE and evaluate if a switch to Cloud makes sense.
+
+## My customer does not have an active champion:
+
+- Customers as of Jan 1, 2023 without an active champion have been identified and are being paired with exec sponsors to help reengage.
+- If a customer falls into this category after the fact, mark the customer as a Red Account via the Account - Red Account? trait on the Vitally account. This will trigger a thread in #red-accounts so that we can strategize on a plan to identify and build new champions.
+- Champions are customer points of contact who will advocate on behalf of sourcegraph. To build champions it is important to understand what an indivduals goal with sourcegraph might be, and help them realize it. If you are able to make a customer contact successful they will be more likely to help you achieve your goals with that company.
+
+## My customer does not have an active buyer that we have access to:
+
+- Sync as an account team and escalate as needed to get Sourcegraph exec help rebuilding that connection.
+- Work with your champion to collect internal wins that came from using Sourcegraph along with feedback.
+- Create a QBR style presentation for your champion showing that value so that they can present Sourcegraph internally and gain support for their initiatives.
+
+## My customer has not been contacted in X number of days:
+
+- Never reach out just to reach out, always work to bring something of value to the customer. This could be a new feature we have released, a new blog post or piece of collateral we’ve created, or some piece of industry news (Log4J) that you believe might affect them PAIRED with some guidance from Sourcegraph.
+
+## My customer has a low Usage Score:
```

Comment:
We deprecated usage score so go ahead and push this but I'll follow along with an update.  MAU/Seats and DAU/MAU are the two new calculations for health score